links/src/utils/getLinks.ts

```diff
@@ -18,7 +18,7 @@ type Links =
 	| {
 			[uuid: string]: Link;
 	  }
-	| undefined;
+	| ArrayLike<Link>;
```

Comment:
```suggestion
	| undefined;
```

Not so sure about this?